Templates/Android/generated/BaseEntityCollectionReferenceRequest.java.tt

```diff
@@ -27,15 +27,15 @@
         final String requestUrl = getBaseRequest().getRequestUrl().toString();
         final ReferenceRequestBody body = new ReferenceRequestBody(getBaseRequest().getClient().getServiceRoot() + "/<#=prop#>/" + new<#=TypeName(c)#>.id);
         new <#=TypeWithReferencesRequestBuilder(c)#>(requestUrl, getBaseRequest().getClient(), /* Options */ null)
-            .buildRequest()
+            .buildRequest(getBaseRequest().getOptions())
```

Comment:
Is there ever a case when `getBaseRequest()` will return null that we should guard against?